app/music_list.rb

```diff
@@ -0,0 +1,46 @@
+require_relative './helper'
+require_relative './music_album'
+
+module MusicList
+  include Helper
+  def list_music_albums
+    if @music_albums.empty?
+      puts 'You don\'t have any Music Albums.'
+    else
+      @music_albums.each_with_index do |music_album, index|
+        # rubocop:disable Layout/LineLength
```

Comment:
It would be better to use the Print method instead of Puts. That way you can put your code in multi lines.